diff --git a/node_modules/@bugsnag/cli/bugsnag-cli-wrapper.d.ts b/node_modules/@bugsnag/cli/bugsnag-cli-wrapper.d.ts
index 3d83f9d..82884cc 100644
--- a/node_modules/@bugsnag/cli/bugsnag-cli-wrapper.d.ts
+++ b/node_modules/@bugsnag/cli/bugsnag-cli-wrapper.d.ts
@@ -1,5 +1,5 @@
 declare module '@bugsnag/cli' {
-    interface BugsnagCreateBuildOptions {
+    interface BaseOptions {
         apiKey?: string
         dryRun?: boolean
         logLevel?: string
@@ -11,7 +11,23 @@ declare module '@bugsnag/cli' {
         timeout?: number
     }
 
-    interface BugsnagUploadReactNativeOptions {
+    export interface BugsnagCreateBuildOptions extends BaseOptions {
+        autoAssignRelease?: boolean
+        buildApiRootUrl?: string
+        builderName?: string
+        metadata?: object
+        provider?: string
+        releaseStage?: string
+        repository?: string
+        revision?: string
+        versionName?: string
+        androidAab?: string
+        appManifest?: string
+        versionCode?: string
+        bundleVersion?: string
+    } 
+
+    export interface BugsnagUploadReactNativeOptions extends BaseOptions {
         bundle?: string
         codeBundleId?: string
         sourceMap?: string
@@ -25,7 +41,7 @@ declare module '@bugsnag/cli' {
         iosXcodeProject?: string
     }
 
-    interface BugsnagUploadiOSOptions {
+    export interface BugsnagUploadiOSOptions extends BaseOptions {
         bundle?: string
         codeBundleId?: string
         sourceMap?: string
@@ -36,7 +52,7 @@ declare module '@bugsnag/cli' {
         xcodeProject?: string
     }
 
-    interface BugsnagUploadAndroidOptions {
+    export interface BugsnagUploadAndroidOptions extends BaseOptions {
         bundle?: string
         codeBundleId?: string
         sourceMap?: string
@@ -46,7 +62,7 @@ declare module '@bugsnag/cli' {
         versionCode?: string
     }
 
-    interface BugsnagUploadJsOptions {
+    export interface BugsnagUploadJsOptions extends BaseOptions {
         baseUrl?: string
         bundle?: string
         bundleUrl?: string
